src/pages/Organizations/organizationInventory.js

```diff
@@ -28,15 +32,40 @@ const AddItemButton = getMyAgent(({agent, loading, error}) => {
     } else {
         for (let i = 0; i < agent.agentRelationships.length; i++) {
             if(agent.agentRelationships[i].object.id === orgId){
-                return <CreateInventoryItem orgId={orgId}/>
+                connected = true;
             }
-
         }
         return <div/>
     }
 
 });
 
+class AddItemButton extends React.Component {
```

Comment:
Why are these whole classes?

Reply:
Removed

Reply:
Also resolved your empty string issue with a commit(not proud of it, but it works with graphql)